<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordChange</class>
 <widget class="QWidget" name="PasswordChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 253, 255);
</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>491</width>
     <height>172</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>50</number>
    </property>
    <property name="rightMargin">
     <number>50</number>
    </property>
    <item>
     <widget class="QLabel" name="oldPasswordLbl">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Old Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="oldPasswordInput">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="newPasswordLbl">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>New Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="newPasswordInput">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="confirmPasswordLbl">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Confirm Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="confirmPasswordInput">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>491</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>100</number>
    </property>
    <property name="rightMargin">
     <number>100</number>
    </property>
    <item>
     <widget class="QPushButton" name="passwordChangeBtn">
      <property name="font">
       <font>
        <family>Bahnschrift SemiBold</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(197, 198, 200);</string>
      </property>
      <property name="text">
       <string>Change password</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
